src/script/components/userDevices/SelfFingerprint.tsx

```diff
@@ -45,17 +45,18 @@ const SelfFingerprint: React.FC<SelfFingerprintProps> = ({
   noPadding,
   clientState = container.resolve(ClientState),
 }) => {
-  const fingerprintLocal = useMemo<string>(
-    () => cryptographyRepository.getLocalFingerprint(),
-    [cryptographyRepository],
-  );
+  const [localFingerprint, setLocalFingerprint] = useState<string>('');
+  useEffect(() => {
+    cryptographyRepository.getLocalFingerprint().then(fingerprint => setLocalFingerprint(fingerprint));
```

Comment:
We can shorten it a bit 😅 
```suggestion
    cryptographyRepository.getLocalFingerprint().then(setLocalFingerprint);
```